components/browse-decks/BrowseDecks.js

```diff
@@ -28,7 +27,7 @@ const DeckWrapper = styled.div`
   @media screen and (max-width: ${screenSizes.smallPhone.max}) {
     justify-content: center;
     align-items: center;
-    margin-bottom: 20px;
+    padding-bottom: 20px;
```

Comment:
Is this necessary?

Reply:
Nope but I just thought that it didn't matter to change it back